libqtile/backend/x11/core.py

```diff
@@ -600,25 +601,60 @@ def handle_ClientMessage(self, event) -> None:  # noqa: N802
             except IndexError:
                 logger.debug("Invalid desktop index: %s" % index)
 
+    def fake_KeyPress(self, keycode, needs_release=True) -> None:
+        xtest = self.conn.conn(xcffib.xtest.key)
+        # The key needs to be released as it was previously pressed
+        if needs_release:
+            xtest.FakeInput(3, keycode, xcffib.xproto.Time.CurrentTime, 0, 0, 0, 0)
```

Comment:
Note to self, in `FakeInput` don't input all zeros but use the event settings. For DeviceId, we need to pass 0 though (last argument).